ArduCopter/version.cpp

```diff
@@ -0,0 +1,16 @@
+#include "version.h"
+
+#include "GCS_MAVLink/GCS.h" // for FIRMWARE_VERSION_TYPE_DEV and friends
```

Comment:
Removed, along with the include since that's now in version.h via `AP_FWVersion`